README.md

```diff
@@ -168,6 +168,10 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
+#### Configuring GitHub Actions
+- On your repository fork, create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `ACCESS_TOKEN` under the <b>Settings/Secrets</b> tab
+- Using the Personal Access Token you placed in the `.env` file earlier create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `OPEN_SOURCE_TOKEN` where the value matches the token value from the `.env` file in your local workspace.
```

Comment:
Also add an line to docs on enabling actions for their repository.

Reply:
I removed line 172 since we defaulted to using `GITHUB_TOKEN`.  I found the JamesIves documentation to explain well enough if this isn't working how to add the `ACCESS_TOKEN`.

Added docs link to Actions Workflow